docs/.sphinx/requirements.in

```diff
@@ -0,0 +1 @@
+git+https://github.com/RadeonOpenCompute/rocm-docs-core.git
```

Comment:
When pulling from a git repo, it should specify a tagged version for stability.

Reply:
We chose not to version due to the rapid development of rocm-docs-core. I think we shouldn't version this until we are ready with a beta release. I will suggest we revert the fix and keep it unversioned. The rate of churn on rocm-docs-core will require daily PRs to keep the projects updated. Once we stabilize rocm-docs-core, I fully support using versions.

Reply:
I believe the timeline for pinning rocm-docs-core should be within a month.

Reply:
The tag should be there so that builds in production are stable.  Developers can override that locally for rapid development.

I still recommend that the dependency is tagged.

Reply:
I know we need to get this tagged. But documentation is not in production yet (we have a massive beta in the background). I just ask for one more week.